apps/site/test/site/content_rewriters/liquid_objects/fare_test.exs

```diff
@@ -122,6 +122,20 @@ defmodule Site.ContentRewriters.LiquidObjects.FareTest do
       assert fare_request("ferry:month") == {:ok, "$80.00 – $329.00"}
     end
 
+    test "handles :ferry:month:reduced" do
+      assert [
+               mode: :ferry,
+               reduced: :any,
+               duration: :month
+             ]
+             |> Repo.all()
+             |> fare_result(:ferry) == "$30.00"
+
+      assert fare_request("ferry_inner_harbor:month:reduced") == {:ok, "$30.00"}
+      refute fare_request("ferry_cross_harbor:month:reduced") == {:ok, "$30.00"}
+      assert fare_request("ferry:month:reduced") == {:ok, "$30.00"}
```

Comment:
I wonder if this is reasonable output for `ferry:month:reduced` even though it's only valid on a certain type of ferry 🤔 cc @amaisano

Reply:
I see what you're saying. The alliterative would be to have either no match (_require_ a ferry_sub_type) or require additional context by the content team for the surrounding text. I would bring it up with Jess or Katrina or Avery and see what they think.

Other alternative ideas besides my suggestions above?